.github/workflows/ci.yml

```
@@ -0,0 +1,40 @@
+  push:
+    branches-ignore:
+      - main
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  quick-test:
+    runs-on: ubuntu-latest
+    
+    steps:
+    - name: Checkout
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)